.travis.yml

```diff
@@ -60,21 +57,12 @@ jobs:
         - echo CONFIG_HAVE_AGENT=n > src/arch/xtensa/configs/override/no-agent.config
         - ./scripts/docker-run.sh ./scripts/xtensa-build-all.sh -o no-agent -r $PLATFORM
         - ./scripts/docker-qemu.sh ../sof.git/scripts/qemu-check.sh $PLATFORM
-      env: PLATFORM='byt cht'
+      env: PLATFORM='icl'
```

Comment:
After whole series applied, "icl" is still left here.

Reply:
Don't touch `.travis.yml`, it has not been used since we moved to Github Actions. There's another PR to finalize the transfer to Github Actions and remove it.